drivers/SmartThings/zwave-switch/profiles/zooz-zen-30-dimmer-relay.yml

```diff
@@ -0,0 +1,280 @@
+    preferenceType: integer
+    definition:
+      minimum: 0
+      maximum: 65535
+      default: 0
+  - name: "dimmerAutoOnParam"
+    title: "Dimmer Auto Turn-On Timer (Minutes)"
+    description: "Turn on when set time comes"
+    required: false
+    preferenceType: integer
+    definition:
+      minimum: 0
+      maximum: 65535
+      default: 0
+  - name: "dimmerRampRateParam"
+    title: "Dimmer Physical Ramp Rate (Seconds)"
+    description: "Set the ramp rate due to physical speed"
+    required: false
+    preferenceType: integer
+    definition:
+      minimum: 0
+      maximum: 99
+      default: 1
+  - name: "dimmerPaddleRampParam"
+    title: "Dimming Speed when Paddle is Held"
+    description: "Dimming Speed when Paddle is Held"
+    required: false
+    preferenceType: integer
+    definition:
+      minimum: 1
+      maximum: 99
+      default: 4
+  - name: "dimmerMinimumBrightParam"
+    title: "Dimmer Minimum Brightness (%)"
+    description: "Select the minimum level of brightness"
+    required: false
+    preferenceType: integer
+    definition:
+      minimum: 1
+      maximum: 99
+      default: 1
+  - name: "dimmerMaximumBrightParam"
+    title: "Dimmer Maximum Brightness (%)"
+    description: "Select the maximum level of brightness"
+    required: false
+    preferenceType: integer
+    definition:
+      minimum: 1
+      maximum: 99
+      default: 99
+  - name: "dimmerCustomBrightParam"
+    title: "Custom Brightness (%)"
+    description: "Select the custom level of brightness"
+    required: false
+    preferenceType: integer
+    definition:
+      minimum: 0
+      maximum: 99
+      default: 0
+  - name: "dimmerBrightControlParam"
+    title: "Dimmer Brightness Control"
+    description: "Dimmer Brightness Control"
+    required: false
+    preferenceType: integer
+    definition:
+      options:
+        0 : "Double Tap Maximum"
+        1 : "Single Tap Custom"
+        2 : "Single Tap Maximum"
+      default: 0
+  - name: "dimmerDoubleTapFuncParam"
+    title: "Dimmer Double Tap Function"
+    description: "Dimmer Brightness Control"
+    required: false
+    preferenceType: integer
+    definition:
+      options:
+        0 : "Turn on Full Brightness"
+        1 : "Turn on Maximum Brightness"
+      default: 0
+  - name: "dimmerLoadControlParam"
+    title: "Dimmer Load Control"
+    description: "Dimmer Load Control"
+    required: false
+    preferenceType: enumeration
+    definition:
+      options:
+        0 : "Physical Disabled"
+        1 : "Physical / Digital Enabled"
+        2 : "Physical / Digital Disabled"
+      default: 1
+  - name: "dimmerPhysDisBehParam"
+    title: "Dimmer Physical Disabled Behavior"
+    description: "Dimmer Physical Disabled Behavior [FIRMWARE >= 1.05]"
+    required: false
+    preferenceType: enumeration
+    definition:
+      options:
+        0 : "Change Status/LED"
+        1 : "Don't Change Status/LED"
+      default: 0
+  - name: "dimmerNightBrightParam"
```

Comment:
since you've already had to change the names of some of those, I'd prefer if we omitted "Param" from the end of all of them